src/pages/careers.tsx

```diff
@@ -240,30 +209,30 @@ const CareersPage: React.SFC<{}> = () => (
-                        <li>Happy to travel regularly</li>
-                    </ul>
-                </Job>
-                <Job>
-                    <h2>Gitpod Sales Engineer</h2>
+                <ExpandableJob jobTitle="Gitpod Developer Advocate">
+                    <div>
+                        <p>As a Developer Advocate, you will connect with developers around the world, try out new frameworks and technologies, and share what you’ve learned with the world. You will craft great resources to help developers build with Gitpod, including sample projects, blog posts, and documentation. You’ll help grow our developer community, champion the interests of our users and translate their feedback into actionable product insights.</p>
+                        <h4>What you’ll do:</h4>
+                        <ul>
+                            <li>Create momentum and drive adoption of Gitpod through your interactions with developers building with Gitpod</li>
+                            <li>Champion Gitpod in person and online by presenting at meetups, conferences, writing technical tutorials, publishing articles and videos</li>
+                            <li>Build inspiring demos, sample apps, and continuously improve the developer experience of Gitpod</li>
+                            <li>Identify strategic partnership opportunities to grow our developer community</li>
+                            <li>Influence the direction of Gitpod’s development by gathering insights from our developer community</li>
+                            <li>Engage with developers to help them get the most out of Gitpod</li>
+                        </ul>
+                        <h4>What we’re looking for:</h4>
+                        <ul>
+                            <li>Creative, empathetic, communicative, and clever</li>
+                            <li>Confident and comfortable presenting to different types of audiences, at meetups or conferences</li>
+                            <li>Adept at communicating technically complicated topics simply and elegantly</li>
+                            <li>Hands-on developer who is comfortable with full-stack development</li>
+                            <li>Intellectually curious, with great problem-solving skills</li>
+                            <li>Happy to travel regularly</li>
+                        </ul>
+                    </div>
+                </ExpandableJob>
+                <ExpandableJob jobTitle='Gitpod Sales Engineer'>
                     <p>As a Sales Engineer, you will take control of our inbound marketing, and leverage Gitpod to help software organizations build better software faster. You will engage with Enterprises and Universities around the globe, understand the way they build software and identify needs and bottlenecks, and build Gitpod-based solutions that make developers happier and more productive.</p>
```

Comment:
Just had an improvement idea, could you please include it in your PR?

```suggestion
                    <p>As a Sales Engineer, you will take control of our inbound marketing, and leverage Gitpod to help software organizations build better software faster. You will engage with Enterprises and Universities around the globe, understand the way they build software and identify their needs and bottlenecks, then design Gitpod-based solutions to make their developers happy and more productive.</p>
```